Signal/src/util/OWSBackup.h

```diff
@@ -4,71 +4,33 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-extern NSString *const OWSBackup_FileExtension;
-
-@protocol OWSBackupDelegate <NSObject>
-
-- (void)backupStateDidChange;
-
-- (void)backupProgressDidChange;
-
-@end
-
-#pragma mark -
+extern NSString *const NSNotificationNameBackupStateDidChange;
 
 typedef NS_ENUM(NSUInteger, OWSBackupState) {
+    // Has never backed up, not trying to backup yet.
+    OWSBackupState_Idle = 0,
+    // Backing up.
     OWSBackupState_InProgress,
-    OWSBackupState_Cancelled,
-    OWSBackupState_Complete,
+    // Last backup failed.
```

Comment:
These are the proposed states to surface to the user in the "app settings > backup" view.  Allows users to confirm that they have a complete backup before migrating devices.